testing/format_trace.sh

```diff
@@ -0,0 +1,55 @@
+if [ $# -eq 0 ]; then
+  echo "Usage: $0 filename"
+  exit 1
+fi
+
+filename="$1"
+target_line="  {"
```

Comment:
Maybe `begin_target_line` is a better name here? Or is it? Is this the beginning of the scope we are interested in? 

And what if there is a nested object in the JSON file? Will the number of spaces in this var be key for the checks later?

Reply:
Done.

Nested objects are not a problem because the json is parsed through jq which will leave the nested objects with a longer indent.